app_dart/lib/src/service/scheduler.dart

```diff
@@ -241,7 +241,8 @@ class Scheduler {
   /// and that do not have any dependencies.
   List<Target> getPreSubmitTargets(Commit commit, SchedulerConfig config) {
     // Filter targets to only those run in presubmit.
-    final Iterable<Target> presubmitTargets = config.targets.where((Target target) => target.presubmit);
+    final Iterable<Target> presubmitTargets =
+        config.targets.where((Target target) => target.presubmit && !target.bringup);
```

Comment:
I have been thinking about whether we need to distinguish pre submit and post submit regarding `bringup`.
With this change here, if a test is `bringup: true`, it will not run in presubmit, but run in post submit without affecting the tree.
When it comes to flakiness in presubmit, I would be interested in running presubmit tests as well, but without affecting the pre-submit tree. A separate control sounds fit here. Does that make sense?
This is based on the assumption that sometimes for some reason, a test maybe flaky in presubmit, but the same test may not be flaky in post submit. I guess that's why we also have the [pre-submit flaky label](https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3A%22team%3A+presubmit+flakes%22).
If the flakiness status of the same test is same for both pre-submit and post-submit, then we may not need to separately address the flaky issues for presubmit. I still need to look into the data to see if this is necessary.

Reply:
This is necessary from the LUCI config side, otherwise it's not possible to add new presubmit targets.

We could enable running bringup presubmit builds, but not updating the check. However, we need to validate this doesn't impact SLOs.

My thought is if presubmit and postsubmit infra are the same, we shouldn't spend extra capacity on the presubmit runs.